client/src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,29 @@
+.nav-bar {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.nav-icons {
+    padding: 0px 10px;
+}
+
+.desktop {
```

Comment:
Desktop what? Can you use a more descriptive classname. Maybe `desktop-style` looking for something that will let another developer know what this class is doing if they can't ask you.

Reply:
Added -style to desktop and mobile